ibverbs/src/lib.rs

```diff
@@ -267,7 +261,10 @@ impl<'devlist> Device<'devlist> {
     pub fn index(&self) -> io::Result<i32> {
         let idx = unsafe { ffi::ibv_get_device_index(*self.0) };
         if idx == -1 {
-            Err(io::Error::new(io::ErrorKind::Unsupported, "device index not known"))
+            Err(io::Error::new(
+                io::ErrorKind::Unsupported,
+                "device index not known",
+            ))
```

Comment:
Is it possible I have different rustfmt settings? And should these be specified in a config at the top of the repo?

Reply:
You probably have a newer version of rustfmt than I had when I wrote that. Totally fine :)